packages/dialect-react-ui/components/NotificationsButton/index.tsx

```diff
@@ -81,6 +82,17 @@ function WrappedNotificationsButton(
   );
   useEffect(() => setRpcUrl(props.rpcUrl || null), [props.rpcUrl, setRpcUrl]);
 
+  const isMobile = useMobile();
+
+  useEffect(() => {
+    // Prevent scrolling of backdrop content on mobile
+    document.documentElement.classList[open && isMobile ? 'add' : 'remove'](
+      'dt-overflow-hidden',
+      'dt-static',
+      'sm:overflow-auto'
```

Comment:
I think you're missing a dt- prefix here

Reply:
nice catch, thanks!